argus/backend/controller/api.py

```diff
@@ -253,6 +255,27 @@ def release_schedules_delete():
     })
 
 
+@bp.route("/release/schedules/update", methods=["POST"])
+@api_login_required
+def release_schedule_update():
+    payload = get_payload(request)
+    req = ScheduleUpdateRequest(**payload)
+    service = ArgusService()
+    update_result = service.update_schedule(
```

Comment:
why not using `ScheduleUpdateRequest` as input parameter to `update_schedule` method?
Also `ScheduleUpdateRequest` could convert to specific types, so later it's not needed to convert to sets and UUID's in method code.

Reply:
Since default `@dataclass` can't do that I propose we leave it as is until later refactor

Reply:
ok